build.gradle

```diff
@@ -30,6 +30,8 @@ if (!repositories) {
 apply plugin: "io.servicetalk.servicetalk-gradle-plugin-internal-root"
 
 task validateLocalDocSite(type: Exec) {
+  group 'Documentation'
```

Comment:
This change is unrelated to the debugging utilities except that it was needed to validate the documentation change and did not appear in the Documentation tasks list.